MADRaS/envs/gym_madras_v2.py

```diff
@@ -256,7 +257,7 @@ def increment_step(self):
             self.step_num += 1
 
 
-class MadrasEnv(gym.Env):
+class MadrasEnv(gym.Env, MultiAgentEnv):
```

Comment:
This makes MADRaS inherently dependent on RLLib. Do we want that? Such a tight integration with RLLib may result in incompatibility with other RL Frameworks like RLPyT. I was thinking of a derived class of MadrasEnv that adds `MultiAgentEnv` as a parent.